brianlong/www.validators.app
# 180143125-cache-cluster-stats

#### What's this PR do?
- create report with cluster stats and use it for the views

#### How should this be manually tested?
- run gather rpc daemon and turn on sidekiq
- check if report was created with the name 'report_cluster_stats'
- check if cluster stats view works as expected

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/180143125

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
